src/VTK/VtkGrid2DRepresentationCells.cxx

```diff
@@ -84,23 +83,19 @@ void VtkGrid2DRepresentationCells::createOutput(const std::string & uuid)
 			double YIOffset = supportingGrid->getYIOffsetInGlobalCrs();
 			double YJOffset = supportingGrid->getYJOffsetInGlobalCrs();
 			std::vector<unsigned int> ptBlank;
-			for (unsigned int j = 0; j < nbNodeJ; ++j)
-			{
-				for (unsigned int i = 0; i < nbNodeI; ++i)
-				{
+			for (unsigned int j = 0; j < nbNodeJ; ++j) {
+				for (unsigned int i = 0; i < nbNodeI; ++i) {
 					unsigned int ptId = i + j * nbNodeI;
-					if (vtkMath::IsNan(z[ptId]))
-					{
+					if (vtkMath::IsNan(z[ptId])) {

```

Comment:
I would rather use the standard isNaN function than the vtk one : https://en.cppreference.com/w/cpp/numeric/math/isnan